profiling/roctx-connector/kp_roctx_connector.cpp

```diff
@@ -14,7 +14,11 @@
 //
 //@HEADER
 
-#include <roctx.h>
+#ifdef KP_ROCTXCONNECTOR_HAVE_ROCPROFILER_SDK_ROCTX
```

Comment:
Why do we need to define our own macro?
Does ROCprofiler-SDK not define anything we could leverage?
Could't we just use an `__has_include` expression?

Reply:
OK. The `CMake` `config` provided by `rocprofiler-sdk-roctx` doesn't seem to set a compiler definition. I've followed your suggestion of using `__has_include`.